src/ucp/core/ucp_proxy_ep.h

```diff
@@ -24,11 +24,12 @@
  * TODO make sure it works with err handling and print_ucp_info
  */
 typedef struct ucp_proxy_ep {
-    uct_ep_t                  super;    /**< Derived from uct_ep */
-    uct_iface_t               iface;    /**< Embedded stub interface */
-    ucp_ep_h                  ucp_ep;   /**< Pointer to UCP endpoint */
-    uct_ep_h                  uct_ep;   /**< Underlying transport endpoint */
-    int                       is_owner; /**< Is uct_ep owned by this proxy ep */
+    uct_ep_t                  super;     /**< Derived from uct_ep */
+    uct_iface_t               iface;     /**< Embedded stub interface */
+    ucp_ep_h                  ucp_ep;    /**< Pointer to UCP endpoint */
+    uct_ep_h                  uct_ep;    /**< Underlying transport endpoint */
+    int                       is_owner;  /**< Is uct_ep owned by this proxy ep */
+    ucp_rsc_index_t           rsc_index; /**< Index of underlying transport endpoint */
```

Comment:
1. `/**< Resource index of underlying transport endpoint */`
2. can remove some of the spaces before the fields to have more room for comments

Reply:
done